pages/api/v1/schedule/index.ts

```diff
@@ -31,9 +45,10 @@ export default async (
     FROM schedules
     WHERE LeagueID=${+league}
       AND SeasonID=${season.SeasonID}
+      ${type ? `AND Type='${type}'` : ''}
```

Comment:
This part I haven't been able to test since, as mentioned, I couldn't get local API endpoint working.

Reply:
Because of how the sql template strings this likely wont work. You will need to probably make the query like:

```js
const search = SQL`
    SELECT *
    FROM schedules
    WHERE LeagueID=${+league}
      AND SeasonID=${season.SeasonID}`

// i think typescript should be able to handle if the value is not one of the ones you provided in the code but if not add a check here for if the type is valid
if (type) { 
  query.append(SQL`AND Type='${type}'`);
}

const schedule = await query(search);
```

This should make it valid since the library uses a specific parsing solution for these strings and doing what you have now will likely break it.